sdk/core/core-util/src/uuidUtils.browser.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Generated Universally Unique Identifier
+ *
+ * @returns RFC4122 v4 UUID.
+ * @internal
```

Comment:
wording and remove @internal:
```suggestion
 * Generate Universally Unique Identifier
 *
 * @returns RFC4122 v4 UUID.
```